Source/Other/CategoryAdditions/SPArrayAdditions.m

```diff
@@ -143,9 +143,16 @@ - (NSArray *)subarrayWithIndexes:(NSIndexSet *)indexes
 
 - (id)objectOrNilAtIndex:(NSUInteger)index
 {
-	if([self count] <= index)
+	if([self count] <= index) // JCS: shouldn't this be just less than?
```

Comment:
Should we remove this completely? It's the same as `safeObjectAtIndex`

Reply:
For the next release I'll sort.